index.html

```diff
@@ -47,31 +47,34 @@
           </ul>
         </div>
 
-        <button>Donate Now</button>
+        <button class='donate'>Donate Now</button>
 
-    </div>
+      </header>
 
     <div class="content">
-        <div class="main">
+        <main class="main">
           <div class="alert">
             You've helped us give <span class="text-highlight">72</span> bikes
             to refugees and asylum seekers so far. <strong>Thank you!</strong>
           </div>
 
           <div class="hero">
-            <h1>Bikes for Refugees</h1>
+            <div>
+              <h1>Bikes for Refugees</h1>
             <p>
               Providing donated bikes and accessories to refugees and asylum
               seekers in Scotland.
             </p>
-            <button>Donate a bike today</button>
-            <button>Volunteer</button>
+            <button class= ' donate'>Donate a bike today</button>
```

Comment:
There is an extra space here that you don't need. Remember to pay attention to the styling of your code and try to keep it consistent where you can.
```suggestion
            <button class= 'donate'>Donate a bike today</button>
```